package.json

```diff
@@ -47,6 +51,8 @@
     "babel-preset-stage-0": "^6.24.1",
     "camelcase": "^4.1.0",
     "codecov": "^2.3.0",
+    "enzyme": "^3.1.0",
+    "enzyme-adapter-react-15": "^1.0.2",
```

Comment:
Is this required for enzyme now? Also, we should probably go ahead and move this repo to React 16 in another PR

Reply:
Yeah, this is an enzyme 3 thing.  Requires setup with an adapter now.

Reply:
What happen if we move to React16 to the other app?

Reply:
I don't suspect using different versions of react here vs in our apps will really make a difference, unless we add react 16 specific things to the design-system, like returning arrays of elements, which we probably shouldn't do in the near term